tests/test_openvpn.py

```diff
@@ -19,6 +20,12 @@ def test_parse(self):
         self.assertEqual(p.version, '1')
         self.assertEqual(p.metric, 'static')
 
+    def test_parse_undef(self):
+        p = OpenvpnParser(links2u)
+        self.assertIsInstance(p.graph, networkx.Graph)
+        self.assertEqual(p.version, '1')
+        self.assertEqual(p.metric, 'static')
```

Comment:
I think here we should add:
```
self.assertEqual(len(data['nodes']), 0)
self.assertEqual(len(data['links']), 0)
```
That's what I'd expect, do these assertions pass?